.github/workflows/markdown_link_checker.yml

```diff
@@ -6,5 +6,5 @@ jobs:
   markdown-link-check:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@master
+    - uses: actions/checkout@v2
```

Comment:
Not sure what the difference between both is but probably a good idea to use a released version.